### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by AlaaShaheen2**
In this project I :
first open a repo 
added a linters
added an html file 
added a css file 
added a test.md file and ignore it 
and finally open this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,56 @@
...
+Give a ⭐️ if you like this project!
+Acknowledgments
+
+    Hat tip to anyone whose code was used
+    Inspiration
+    etc
+
+📝 License
+
+This project is MIT licensed.
```
Comment (sumairq) [inline README.md]: - [ ] Kindly customize the Readme template , It seems that you forgot to include the specific details within the placeholder section in this template. Kindly Edit it and make it your own. Cheers 😃

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (sumairq) [inline .gitignore:1]: - [ ] Kindly add the node_modules directory to the `.gitignore` file. 
It is always a good idea to add the node_modules directory to the `.gitignore` file in all your projects as it is part of the standard practice.
This way you can avoid putting possibly hundreds of MB of dependencies in your repository. 😄

**Review by sumairq — CHANGES_REQUESTED** [review]
### Hi @AlaaShaheen2 , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good commit history
✔️ Used correct Github flow
✔️ Descriptive PR

## Required Changes ♻️
- [ ] Let's make sure linters are configured properly, as the tests are not running in this case.  refer to the microverse [linter configuration guidelines](https://github.com/microverseinc/linters-config)
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AlaaShaheen2** [comment]
@sumairq  i did the changes

File: .gitignore (hunk 2 above)
Comment (mugishasam123) [reply .gitignore:1]: - [ ] Kindly can you add node_modules directory in `.gitignore` file as requested by previous reviwer

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @AlaaShaheen2  👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Good PR summary
✔️ Descriptive PR title


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ Optional] I would recommend you correct all linter errors because they help in cleaning up the code and keep your project more maintainable and professional.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AlaaShaheen2** [comment]
@mugishasam123 I tried but I don't know how can you help?
I made the Required Changes 
I left the optional changes I did some of them

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @AlaaShaheen2 

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**